hazelcast/src/main/java/com/hazelcast/collection/impl/queue/QueueProxyImpl.java

```diff
@@ -233,10 +233,6 @@ public boolean retainAll(Collection<?> objects) {
 
     @Override
     public String toString() {
-        final StringBuilder sb = new StringBuilder();
-        sb.append("IQueue");
-        sb.append("{name='").append(name).append('\'');
-        sb.append('}');
-        return sb.toString();
+        return "IQueue{name='" + name + '\'' + '}';
```

Comment:
I'm not sure about merging this to a single String: `'\'' + '}'`
Performance should be better with two chars, WDYT?